edx_proctoring/views.py

```diff
@@ -862,11 +862,19 @@ def make_review(self, attempt, data, backend=None):
         if review.review_status in SoftwareSecureReviewStatus.notify_support_for_status:
             instructor_service = get_runtime_service('instructor')
             if instructor_service:
+                course_id = attempt['proctored_exam']['course_id']
+                exam_id = attempt['proctored_exam']['id']
+                review_url = request.build_absolute_uri(
+                    u'{}?attempt={}'.format(
```

Comment:
Where does this URL get routed? I don't see a URLConf ending in `?attempt` anywhere.

Reply:
`?attempt=` is a query parameter added onto the url created by `reverse('edx_proctoring:instructor_dashboard_exam', args=[course_id, exam_id])`